### PR TITLE
Added updated logo to repo and changed header image in readme to use it

### PR DESCRIPTION
**Pull request by mrshll1001**
This PR adds a new file into the repo, `360standard-color.png`, taken from the [branding guidelines logo page](https://cdn.threesixtygiving.org/components/detail/logos.html) and updates the README file to point towards it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
 360Giving Standard
 ========
 
 360Giving aims to help UK grant makers and philanthropists to publish their grant information online in an easy to use way.  We are motivated by a commitment to the value of open data and strategic funding.
 
-The **360Giving Data Standard** provides a common approach for publishing grant data. 
+The **360Giving Data Standard** provides a common approach for publishing grant data.
 
 For more about the overall project, visit the main [360Giving website](http://threesixtygiving.com/about/).
```
Comment (KDuerden) [inline README.md:10]: update to current website url: https://www.threesixtygiving.org/

Comment (KDuerden) [reply README.md:10]: also be good to update url on row 12 from http://www.threesixtygiving.org/standard to https://standard.threesixtygiving.org/en/latest/

File: README.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-![360Giving Logo](http://openphilanthropy.files.wordpress.com/2014/02/cropped-360givinglogo-2010-size.jpg)
+![360Giving Logo](./360standard-color.png)
 
 360Giving Standard
 ========
 
 360Giving aims to help UK grant makers and philanthropists to publish their grant information online in an easy to use way.  We are motivated by a commitment to the value of open data and strategic funding.
```
Comment (KDuerden) [inline README.md:6]: update text to align with current mission statement: 

360Giving's mission is to help UK funders publish open, standardised grants data, and empower people to use it to improve charitable giving

**Review by KDuerden — CHANGES_REQUESTED** [review]
Thanks. That's fine but I've added some comment to update the website links and updating the statement about 360Giving to match our mission statement.

**Comment by Bjwebb** [comment]
If we add the logo to the repo, we need to add a sentence to the license file, as we do for CoVE:

> Note: This license doesn't apply to the 360Giving theme and related assets (such as the logo).

**Comment by mrshll1001** [comment]
I've updated the PR and made the changes noted. Let me know if there's anything else :-)

**Review by mariongalley — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+360Giving's mission is to help UK funders publish open, standardised grants data, and empower people to use it to improve charitable giving.
 
-The **360Giving Data Standard** provides a common approach for publishing grant data. 
+The **360Giving Data Standard** provides a common approach for publishing grant data.
 
-For more about the overall project, visit the main [360Giving website](http://threesixtygiving.com/about/).
+For more about the overall project, visit the main [360Giving website](http://threesixtygiving.org/).
 
-Full documentation of the data standard is provided at [http://www.threesixtygiving.org/standard](http://www.threesixtygiving.org/standard/)
+Full documentation of the data standard is provided at [http://www.threesixtygiving.org/standard](http://www.threesixtygiving.org/standard/en/latest)
```
Comment (KDuerden) [inline README.md:12]: This url is wrong, it needs to be https://standard.threesixtygiving.org/en/latest/

File: README.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
 ========
 
-360Giving aims to help UK grant makers and philanthropists to publish their grant information online in an easy to use way.  We are motivated by a commitment to the value of open data and strategic funding.
+360Giving's mission is to help UK funders publish open, standardised grants data, and empower people to use it to improve charitable giving.
 
-The **360Giving Data Standard** provides a common approach for publishing grant data. 
+The **360Giving Data Standard** provides a common approach for publishing grant data.
 
-For more about the overall project, visit the main [360Giving website](http://threesixtygiving.com/about/).
+For more about the overall project, visit the main [360Giving website](http://threesixtygiving.org/).
```
Comment (KDuerden) [inline README.md:10]: It would be good to link to the secure url version - https://www.threesixtygiving.org/

File: LICENSE.md [hunk 5]
```diff
@@ -1 +1,5 @@
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+<p>Note: These licenses do not apply to the 360Giving theme and related assets (such as the logo).</p>
+
+<p>
+  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (KDuerden) [inline LICENSE.md:4]: It would be good to link to the secure version in these links - https://creativecommons.org/licenses/by/4.0/

**Review by KDuerden — CHANGES_REQUESTED** [review]
I've added comments on three areas that I think need updating - to use the secure version of each website and to correct the link to the Standard website which is currently incorrect

**Comment by mrshll1001** [comment]
Cheers @KDuerden, I've amended as per suggestions. Agree we should promote https where possible and explicitly point to the latest branch of the standard docs!

**Review by KDuerden — APPROVED** [review]
I've reviewed the changes and I'm happy that the updates and confirm that this is a non-normative change, so doesn't need to go through the Standard governance processes.

**Pull request merged**